lib/Backends/CPU/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ endif()
 find_program(LLVM_LINK_BIN
              NAMES
                llvm-link-6.0
+               llvm-link-7
```

Comment:
https://github.com/pytorch/glow/blob/master/CMakeLists.txt#L79

might want to switch order here.